idunn/places/event.py

```diff
@@ -32,7 +32,6 @@ def build_address(self, lang):
         return {
             "name": self.get("placename"),
             "label": self.get("address"),
-            "city": self.get("city"),
```

Comment:
This field `city` was specific to `Event` objects, and does not fit the `Address` model used by other objects.  
It's not used by any application, and the city name is already part of the label. 
So there is no reason to keep it.